.github/workflows/android-e2e.yml

```diff
@@ -3,7 +3,7 @@ name: Android e2e tests
 on: [pull_request]
 jobs:
   android-e2e:
-    runs-on: blacksmith-8vcpu-ubuntu-2204
+    runs-on: blacksmith-16vcpu-ubuntu-2204
```

Comment:
We should remove this change and the telemetry for now since it was just to debug.